core/common/src/main/java/alluxio/client/file/URIStatus.java

```diff
@@ -131,7 +133,9 @@ public int getInAlluxioPercentage() {
   }
 
   /**
-   * @return the epoch time the entity referenced by this uri was last modified, mutable
+   * Gets the time the entity referenced by this URI was last modified.
+   *
+   * @return  the epoch time the entity referenced by this uri was last modified, mutable
```

Comment:
Keep consistency with lower/upper case
```suggestion
   * @return  the epoch time the entity referenced by this URI was last modified, mutable
```